### PR TITLE
Full screen fixed header

### PR DESCRIPTION
**Pull request by sarahkleins**
Adds a fixed header example 
screenshot 2017-01-19 18 53 11

### CODE REVIEW

File: site/examples/fixed_header.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const fixed_header = `<div class='bg-darken10 viewport-half'>
+<!-- In practice, it makes sense to use this for the container instead:
```
Comment (samanpwbb) [inline site/examples/fixed_header.js]: The fixed and not-fixed versions of this header should be as similar as possible. You could use:

`<div class='absolute top right bottom left scroll-auto'></div>`

as an alternative to 

`<div class='fixed top right bottom left scroll-auto'></div>`

Only one class different, and should communicate the idea just as well.

---

Also I don't think you need `scroll-auto` on this example.

File: site/examples/fixed_header.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const fixed_header = `<div class='bg-darken10 viewport-half'>
+<!-- in practice, the header container should use the class "fixed" instead of "absolute" -->
+    <div class='relative'>
```
Comment (davidtheclark) [inline site/examples/fixed_header.js:6]: We've been using 2 spaces for indentation everywhere else.

**Review by davidtheclark — COMMENTED** [review]
@sarahkleins I think the implementation is good here, but I'm wondering what purpose the example serves. All that it's doing right now is showing `absolute` in action. Are there certain aspects of a fixed header that you consider difficult or non-intuitive?

**Comment by davidtheclark** [comment]
@sarahkleins Continuing my comment above ... Maybe it needs some long, scrolling content (which also has a top margin to offset the heading)? Then we could see the stickiness in action.

**Comment by sarahkleins** [comment]
@tristen @davidtheclark This example was at @tristen 's suggestion to replace this guy: http://bl.ocks.org/tristen/768ac555d78b7b7123791580b2f21b90

I can also add more content, like a long scrolling sidebar offset w/the header? Thoughts @tristen ?

**Comment by tristen** [comment]
Not sure how important this layout is @sarahkleins. I don't think there should be any scrollable content on the main container as it was designed to be a full page app. The origin of this layout came from a support request(s) where a user wanted a full page map and header with no overflow (cc @geografa as I think we talked about it in the past?)

**Comment by sarahkleins** [comment]
Checking up on this - any consensus on if we need this example? If not, I can close the PR.

cc @tristen @samanpwbb @davidtheclark

**Comment by tristen** [comment]
@sarahkleins Let's maybe pause on this one? If it's a common layout as we start to use Assembly more we can return to it.

**Pull request closed without merging**